h11/tests/test_helpers.py

```diff
@@ -1,12 +1,21 @@
-from .helpers import *
+from .._events import (
+    ConnectionClosed,
+    Data,
+    EndOfMessage,
+    Event,
+    InformationalResponse,
+    Request,
+    Response,
+)
+from .helpers import normalize_data_events
 
 
-def test_normalize_data_events():
+def test_normalize_data_events() -> None:
     assert normalize_data_events(
         [
             Data(data=bytearray(b"1")),
             Data(data=b"2"),
-            Response(status_code=200, headers=[]),
+            Response(status_code=200, headers=[]),  # type: ignore[arg-type]
```

Comment:
I'm curious about these `ignore` cases, since it looks correct to me?

Reply:
I think mypy gets confused about the type of the empty list. There is an open bug I can't find at the moment